pkg/cmd/template/data_values_flags.go

```diff
@@ -40,8 +39,10 @@ type DataValuesFlags struct {
 	InspectSchema  bool
 	SkipValidation bool
 
-	EnvironFunc  func() []string
-	ReadFileFunc func(string) ([]byte, error)
+	EnvironFunc   func() []string
+	ReadFilesFunc func(paths string) ([]*files.File, error)
+
+	files.SymlinkAllowOpts
```

Comment:
since users control the "values pipe" i dont think we should be restricting usage of symlinks for data value ingestion.

Reply:
I had included this more with an understanding of protection/confidence...

That is, safe defaults with the option to do riskier things if you indicate you accept that risk.

We're walking a directory tree that is providing data values... shouldn't we ensure that if we're traversing a symlink that the user intends it?

Also, if our intention is to support traversing directories, we've got more work to do here.

Reply:
Resolution: keep for now.